health/micro-ui-react19/web/packages/css/src/pages/employee/healthdss.scss

```
@@ -1582,71 +1596,155 @@
-    .digit-checkbox-container {
-      margin: 0rem;
+      .digit-tag-wrapper {
+        border-radius: 12px;
+
+        .digit-tag-text::before {
+          content: "";
+          display: inline-block;
+          width: 0.5rem;
+          height: 0.5rem;
+          border-radius: 50%;
+          background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)